cmd/internal/flowmode/cmd_run.go

```diff
@@ -365,8 +365,7 @@ func (fr *flowRun) Run(configFile string) error {
 // getEnabledComponentsFunc returns a function that gets the current enabled components
 func getEnabledComponentsFunc(f *flow.Flow) func() map[string]interface{} {
 	return func() map[string]interface{} {
-		components := f.ListComponents(component.InfoOptions{})
-
+		components := component.GetAllComponents(f, component.InfoOptions{})
```

Comment:
This means usage stats will now contain stats across all modules, which previously wasn't happening 🎉